packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -101,20 +78,21 @@ export interface TrackerProps {
   onStartEdit: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 }
 
-interface FileStatus extends Partial<FileStateProps> {
+export interface FileStatus extends Partial<FileStateProps> {
   percentage?: number;
   uploadTask?: UploadTask;
   fileErrors?: string;
+  name?: string;
+  file?: File;
 }
 
 export type FileStatuses = FileStatus[];
 
+type StatesOfFiles = 'paused' | 'success' | 'error' | null;
 export interface FileStateProps {
-  loading: boolean;
-  success: boolean;
-  error: boolean;
-  paused: boolean;
```

Comment:
All have been deleted and cleaned up. However, something can be in a loading state and paused state at the same time. That's why loading is still available.

Reply:
It feels odd to separate the loading state, maybe we can add a `loading_paused` state?

Reply:
OK, I'll remove the isLoading, and add it to a file state.